mmv1/third_party/terraform/website/docs/r/bigtable_gc_policy.html.markdown

```diff
@@ -66,6 +66,65 @@ resource "google_bigtable_gc_policy" "policy" {
 }
 ```
 
+For complex, nested policies, an optional `gc_rules` field are supported. This field
+conflicts with `mode`, `max_age` and `max_version`. This field is a serialized JSON
+string. Example:
+```hcl
+resource "google_bigtable_instance" "instance" {
+  name = "instance_name"
+
+  cluster {
+    cluster_id = "cid"
+    zone       = "us-central1-b"
+  }
+
+  instance_type = "DEVELOPMENT"
+  deletion_protection = false
+}
+
+resource "google_bigtable_table" "table" {
+  name          = "your-table"
+  instance_name = google_bigtable_instance.instance.id
+
+  column_family {
+    family = "cf1"
+  }
+}
+
+resource "google_bigtable_gc_policy" "policy" {
+  instance_name = google_bigtable_instance.instance.id
+  table         = google_bigtable_table.table.name
+  column_family = "cf1"
+
+  gc_rules = <<EOF
+{
+  "mode": "union",
+  "rules": [
+    {
+      "max_age": "10h"
+    },
+    {
+      "mode": "intersection",
+      "rules": [
+        {
+          "max_age": "2d"
+        },
+        {
+          "max_version": "2"
+        }
+      ]
+    }
+  ]
+}
+EOF
```

Comment:
Tested this locally and got 2 errors:
1. `Error: invalid duration string: 2d`
2. `Error: `max_version` must be a number`

Would you mind adding this to one of the tests? We want to make sure that the examples in our documentation also got tested. So if users copy them in their configs, they will not get any errors. Thanks!